src/mappings/mappingUniquesHandlers.ts

```diff
@@ -0,0 +1,21 @@
+import { SubstrateEvent } from "@subql/types";
+import { UniquesTransfer } from "../types/models";
+
+export async function handleUniquesTransferEvent(event: SubstrateEvent): Promise<void> {
+    logger.debug('uniqueTransferEvent added'  + JSON.stringify(event.toHuman()))
```

Comment:
In the future, we would probably benefit from structured logging (since it's easier to search by specific fields in log systems like Datadog or GCP Logging). Not critical, but something to consider in the future.